sycl/test-e2e/Regression/swizzle_opassign.cpp

```diff
@@ -0,0 +1,78 @@
+// REQUIRES: aspect-usm_shared_allocations
```

Comment:
I think we decided we don't want to be that restrictive in our tests and prefer to re-write to buffers or, at least, device USM.

Reply:
I was not aware, but no complaints!